workspaces/orchestrator/plugins/orchestrator-backend/src/service/DevModeService.ts

```diff
@@ -155,10 +159,17 @@ export class DevModeService {
       '-v',
       `${resourcesAbsPath}:${SONATA_FLOW_RESOURCES_PATH}:Z`,
     );
+    if (!fs.existsSync(persistenceAbsPath)) {
+      fs.mkdirSync(persistenceAbsPath, { recursive: true });
+    }
+    launcherArgs.push(
+      '-v',
+      `${persistenceAbsPath}:${DEFAULT_SONATAFLOW_PERSISTENCE_PATH}:Z`,
+    );
```

Comment:
what if i want to use a custom persistence path? It seems that, if the `persistenceAbsPath` exists then the `DEFAULT_SONATAFLOW_PERSISTENCE_PATH` won't be used in any case. Can you please clarify?